filters/ignore_new_executors.hpp

```diff
@@ -0,0 +1,65 @@
+#include <ctime>
+#include <memory>
+
+#include "mesos/mesos.hpp"
+
+#include "messages/serenity.hpp"
+
+#include "serenity/executor_map.hpp"
+#include "serenity/serenity.hpp"
+#include "serenity/std_wrappers.hpp"
+
+#include "stout/nothing.hpp"
+#include "stout/option.hpp"
+#include "stout/try.hpp"
+
+#ifndef SERENITY_IGNORE_NEW_TASKS_FILTER_HPP
+#define SERENITY_IGNORE_NEW_TASKS_FILTER_HPP
+
+namespace mesos {
+namespace serenity {
+
+
+/**
+ * IgnoreNewExecutorsFilter removes executors from ResourceUsage collection
+ * that run for less time than threshold.
+ * It's purpose is to cut away tasks that are warming up.
+ */
+class IgnoreNewExecutorsFilter : public Consumer<ResourceUsage>,
+                                 public Producer<ResourceUsage> {
+ public:
+  explicit IgnoreNewExecutorsFilter(
+    Consumer<ResourceUsage>* _consumer = nullptr,
+    uint32_t _threshold = DEFAULT_THRESHOLD) :
+      Producer<ResourceUsage>(_consumer),
+      threshold(_threshold),
```

Comment:
Could you please augment this somehow to communicate the time unit you interpret later, either by adding a docblock comment or by renaming the argument to `_thresholdSeconds`?